DISCOVER/02_minimal_measures.md

```diff
@@ -3,11 +3,11 @@
 
 ***As a general rule, we encourage conference organizers to implement all the 🍎 measures and at least one of the ✅ suggestions.***
 
-Making your conference inclusive and encouraging diversity is a breeze!
+Making your conference inclusive and encouraging diversity is a achievable. The measures below are explained in more depth in the other pages of this Cookbook.
```

Comment:
"is a achievable" isn't a complete phrase.

Reply:
I think it’s meant to be
> Making your conference inclusive and encouraging diversity is achievable.

I think it’s a typo and the “a” was just left over? Remove it and it makes more sense to me.

```suggestion
Making your conference inclusive and encouraging diversity is achievable. The measures below are explained in more depth in the other pages of this Cookbook.
```

Reply:
Also this repo is set up with a “preview” action that renders the Jupyter book in the PR, someone with “write” access has to approve the workflow to enable it.

Reply:
Hi, I've made the requested updates. Please let me know if further modifications are needed.
Thanks